sdk/textanalytics/Azure.AI.TextAnalytics/src/RecognizePiiEntitiesOptions.cs

```diff
@@ -25,5 +25,13 @@ public RecognizePiiEntitiesOptions()
         /// For more information see <a href="https://aka.ms/tanerpii"/>.
         /// </summary>
         public PiiEntityDomainType DomainFilter { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value that, if set, indicates the version of the text
+        /// analytics model that will be used to generate the result.  For supported
+        /// model versions, see operation-specific documentation, for example:
+        /// <a href="https://docs.microsoft.com/azure/cognitive-services/text-analytics/how-tos/text-analytics-how-to-sentiment-analysis#model-versioning"/>.
+        /// </summary>
+        public new string ModelVersion { get; set; } = "latest";
```

Comment:
this is going to affect the sync `Pii` calls. The best way to do this will be to set the model version probably in the Transform class when all the `xxParameterTask` objects are created. That way it is specific only to analyze and once the service fixes it, we can remove it